perf/benchmark/run_benchmark_job.sh

```diff
@@ -50,7 +51,7 @@ function setup_metrics() {
     export FORTIO_CLIENT_URL=http://localhost:8080
   fi
   export PROMETHEUS_URL=http://localhost:9090
-  kubectl -n istio-prometheus port-forward svc/istio-prometheus 9090:9090 &>/dev/null &
+  kubectl -n "${PROMETHEUS_NAMESPACE}" port-forward svc/prometheus 9090:9090 &>/dev/null &
```

Comment:
So, when using istioctl, Prometheus is install directly under istio-system namespace?

Reply:
yes